### PR TITLE
Add seed date capability.

### PR DESCRIPTION
**Pull request by kayernyc**
Added seed date capability and sanitization to the adapter.

### CODE REVIEW

File: src/enums/ViewConfiguration.ts [hunk 1]
```diff
@@ -2,23 +2,25 @@ import DateType from './DateType';
...
-  viewType: ViewType;
+  grouped?: boolean;
+  looping?: boolean;
   maxDate?: Date;
   minDate?: Date;
-  continuousScroll?: boolean;
-  looping?: boolean;
-  grouped?: boolean;
+  seedDate?: Date;
+  viewType: ViewType;
```
Comment (deadnancy) [inline src/enums/ViewConfiguration.ts:23]: ❤️  alphabetizing params.

File: src/models/ViewConfigurationAdapter.ts [hunk 2]
```diff
@@ -65,12 +68,17 @@ export default class ViewConfigurationAdapter {
...
+      object.seedDate = ViewConfigurationAdapter.sanitizedDate(seedNumber)
+    }
+
     if (
       // tslint:disable: no-string-literal
       object.dateType !== undefined &&
       [0, 1, 2, 3, 4, 5, 6, 7].includes(object.dateType) &&
       object.viewType !== undefined &&
-      [0, 1, 2, 3, 4, 5, 6, 7].includes(object.viewType)
+      [0, 1].includes(object.viewType)
```
Comment (deadnancy) [inline src/models/ViewConfigurationAdapter.ts:81]: Feels like a lot of `return` statements in this method. Maybe break things up in a future refactor?

File: src/models/datePickerFactory/DatePickerFactory.ts [hunk 3]
```diff
@@ -26,59 +26,63 @@ export default class DatePickerFactory {
...
   private maxDate?: Date;
   private minDate?: Date;
+  private seedDate: Date;
   private grouped?: boolean;
   dateTimeFormat: DateTimeFormat;
   private atomicDateObjectFunction: AtomicDateObjectCreator;
 
   constructor(config: ViewConfiguration) {
-    const { dateType, maxDate, minDate, grouped } = config;
+    const { dateType, maxDate, minDate, grouped, seedDate } = config;
```
Comment (deadnancy) [inline src/models/datePickerFactory/DatePickerFactory.ts:35]: ...but then should horizontal params get alphabetized too? I am conflicted.

**Review by deadnancy — APPROVED** [review]
Looks good to me!

File: src/models/datePickerFactory/DatePickerFactory.ts (hunk 3 above)
Comment (kayernyc) [reply src/models/datePickerFactory/DatePickerFactory.ts:35]: for a future refactor, when min/max are also in there :)

File: src/models/ViewConfigurationAdapter.ts (hunk 2 above)
Comment (kayernyc) [reply src/models/ViewConfigurationAdapter.ts:81]: good call!

**Pull request merged**